spec/takeaway_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'takeaway'
+require 'twilio'
+require 'support/shared_examples_for_twilio'
+
+describe Takeaway do
+
+  let(:takeaway) {described_class.new}
+  let(:dish_klass) {double(new: dish, price: 4)}
+  let(:class_double) {double}
+  let(:dish) {double(class: class_double)}
+
+  context 'The menu' do
+    let(:menu_file) {double(each_line: nil)}
+
+    describe '#menu' do
+      it 'displays the list of dishes' do
+        allow(File).to receive(:read).and_return(menu_file)
+        expect(File).to receive(:read)
+        takeaway.menu
+      end
+    end
+  end
+
+  context 'Ordering dishes' do
+    describe '#take_order' do
+      it 'raises an error when user\'s cost total doesn\'t match the real total' do
```

Comment:
Metrics/LineLength: Line is too long. [83/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)